### DISCUSSION TITLE
輸不起到底是什麼意思？

### DISCUSSION THREAD

**Discussion by harrycoconut** [1101-GS4719-C/Python-Community-GS4719-C#1387] [Q&A]
我最近一直在思考一個問題，輸不起的反義詞是什麼？輸得起？輸得起又是什麼鬼意思？你輸得很開心？我認為當一個人在乎他的比賽或重視他正在做的事時，絕對不會輸得起。
你們覺得輸不起是貶義嗎？我反而認為是一種稱讚，代表你很重視你的比賽，並且竭盡所能想找出你輸在哪裡😌

**Comment by tower20021202** [comment]
我認為輸不起他沒有絕對的反義詞，但通常人們輸不起的情況都是用來形容輸了卻在找藉口來掩蓋自己輸的事實，不願直視自己的弱點，去設法改善自己。

**Comment by shuxian12** [comment]
我覺得輸不起這個詞是本身還是有點點貶義的，某種程度上來講好像會變成得失心太重，因為如果你很重視這個比賽，雖然結果勢必很重要，但如果只看結果的話，好像就真的你不是為了獲得經驗，而是為了名聲而去參加，我自己是覺得過程比結果重要！！

**Comment by 1104030360** [comment]
我認為輸不起一開始是好事，代表有在重視這件事，但是一旦因為輸不起去做一些傷害他人的事那就非常不好了

**Comment by ALAN30359704** [comment]
我認爲輸不起的意思是指說輸了以後，耍不正當手段報復的心態。

**Comment by Tzuhsuannnnn** [comment]
我認為用輸不起的態度去做事不是一件壞事，有這個態度可以讓自己更認真的面對事情，但也要適度的輸不起，在輸掉某個比賽或某件事後，檢討完自己的不足就要起身開始下一件事了，而不是持續沉溺於這個心境。

**Comment by lin157644** [comment]
辭典的定義是口出怨言或不正當報復，我認為對於比賽的重視不一定要用這種方式表達，輸得開心很難，但是不開心之餘，我更傾向立即檢討與思考輸的原因，用言語或其他方式報復只是逃避自我的不足，輸不起一詞作為貶義並沒有什麼問題。
